hirola/front/static/front/js/dashboard.js

```diff
@@ -62,3 +62,18 @@ function openDisplay(value) {
           break;
       }
   }
+
+
+$(document).ready(function () {
+    var sPath = window.location.pathname;
+    var sPage = sPath.substring(sPath.lastIndexOf('/') + 1);
+    console.log(sPage);
```

Comment:
Remove the console statement to prevent printing of items on the console that a user should not view.